catalog/spatial/csw/spatial-csw-action/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/actions/UpdateAction.java

```diff
@@ -11,26 +11,22 @@
  * License is distributed along with this program and can be found at
  * <http://www.gnu.org/licenses/lgpl.html>.
  */
-package org.codice.ddf.spatial.ogc.csw.catalog.common.transaction;
+package org.codice.ddf.spatial.ogc.csw.catalog.actions;
 
-/** Contains attributes shared by the concrete CSW transaction action classes. */
-public abstract class CswAction {
+import ddf.catalog.data.Metacard;
+import java.io.Serializable;
+import java.util.Map;
+import net.opengis.cat.csw.v_2_0_2.QueryConstraintType;
 
-  protected String typeName;
+public interface UpdateAction {
```

Comment:
Could you add some javadoc to this? And should it be marked as experimental?

Reply:
@bdeining are you saying they don't need to marked as experimental?